Orange/widgets/gui.py

```diff
@@ -2293,12 +2295,12 @@ def commit():
     def do_commit():
         nonlocal dirty
         QApplication.setOverrideCursor(QCursor(Qt.WaitCursor))
-        master.unconditional_commit()
+        getattr(master, 'unconditional_' + callback)()
```

Comment:
If the method is given a reference instead of string, this line can be replaced by `callback()`.